.travis.yml

```diff
@@ -11,13 +11,6 @@ before_script:
 script:
 - python3 manage.py test
 deploy:
```

Comment:
this line also needs to be deleted

Reply:
Just the deploy line or all of 
```
script:
- python3 manage.py test
```

Reply:
just the word `deploy` (since it is redundant)

Reply:
Done. Thank you!